packages/radio-group/src/vaadin-radio-button.js

```diff
@@ -4,70 +4,95 @@
 import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /**
- * `<vaadin-radio-button>` is a Web Component for radio buttons.
+ * `<vaadin-radio-button>` is a web component representing a choice in a radio group.
+ * Only one radio button in a group can be selected at the same time.
  *
  * ```html
- * <vaadin-radio-button value="foo">Foo</vaadin-radio-button>
+ * <vaadin-radio-group label="Travel class">
+ *  <vaadin-radio-button value="economy">Economy</vaadin-radio-button>
+ *  <vaadin-radio-button value="business">Business</vaadin-radio-button>
+ *  <vaadin-radio-button value="firstClass">First Class</vaadin-radio-button>
+ * </vaadin-radio-group>
  * ```
  *
  * ### Styling
  *
  * The following shadow DOM parts are available for styling:
  *
- * Part name         | Description
- * ------------------|----------------
- * `radio`           | The radio button element
- * `label`           | The label content element
+ * Part name   | Description
+ * ------------|----------------
+ * `container` | The container element.
+ * `radio`     | The wrapper element that contains slotted `<input type="radio">`.
+ * `label`     | The wrapper element that contains slotted `<label>`.
  *
  * The following state attributes are available for styling:
  *
- * Attribute  | Description | Part name
- * -----------|-------------|------------
- * `disabled`   | Set when the radio button is disabled. | :host
- * `focus-ring` | Set when the radio button is focused using the keyboard. | :host
- * `focused`    | Set when the radio button is focused. | :host
- * `checked`    | Set when the radio button is checked. | :host
- * `empty`      | Set when there is no label provided. | label
+ * Attribute    | Description | Part name
+ * -------------|-------------|------------
+ * `active`     | Set when the radio button is pressed down, either with a pointer or the keyboard. | `:host`
+ * `disabled`   | Set when the radio button is disabled. | `:host`
+ * `focus-ring` | Set when the radio button is focused using the keyboard. | `:host`
+ * `focused`    | Set when the radio button is focused. | `:host`
+ * `checked`    | Set when the radio button is checked. | `:host`
+ * `has-label`  | Set when the radio button has a label. | `:host`
  *
  * See [Styling Components](https://vaadin.com/docs/latest/ds/customization/styling-components) documentation.
  *
- * @fires {Event} change - Fired when the user commits a value change.
  * @fires {CustomEvent} checked-changed - Fired when the `checked` property changes.
  *
  * @extends HTMLElement
- * @mixes ElementMixin
- * @mixes ControlStateMixin
  * @mixes ThemableMixin
- * @mixes GestureEventListeners
- * @element vaadin-radio-button
+ * @mixes ElementMixin
+ * @mixes ActiveMixin
+ * @mixes AriaLabelMixin
+ * @mixes InputSlotMixin
+ * @mixes CheckedMixin
+ * @mixes SlotLabelMixin
+ *
+ * TODO:
+ * // If you change this block, please test manually that radio-button and
```

Comment:
While I believe using the slotted input could fix the issue, we actually no longer have to support iOS 12/13 which means this issue is probably no longer needs taking care of.